vendor/foundation_rails_helper/form_builder.rb

```diff
@@ -0,0 +1,245 @@
+# frozen_string_literal: true
+
+# Copyright (c) 2015 Sébastien Gruhier (http://xilinus.com/)
+# MIT License
+# Vendored copy from https://github.com/sgruhier/foundation_rails_helper/blob/master/lib/foundation_rails_helper/form_builder.rb
+#
+# This file was vendored to work around the Rails 7.1 incompatibility,
+# since the `foundation_rails_helper` gem is no longer maintained.
+#
+# Safe to remove once we drop legacy Foundation CSS support.
```

Comment:
I think this isn't really true :thinking:. The `foundation_rails_helper` name is misleading in our case because the funcionality we use is not related to Foundation.

```suggestion
```